reference-implementation/lib/transform-stream.js

```diff
@@ -325,7 +325,7 @@ function TransformStreamDefaultControllerError(controller, e) {
 
 function TransformStreamDefaultControllerPerformTransform(controller, chunk) {
   const transformPromise = controller._transformAlgorithm(chunk);
-  return transformPromise.catch(r => {
+  return PerformPromiseCatch(transformPromise, r => {
```

Comment:
We can use PerformPromiseThen() here and it will be closer to [the description in the promises guide](https://www.w3.org/2001/tag/doc/promises-guide/#transforming-by).

Reply:
You mean like this?
```javascript
return PerformPromiseThen(transformPromise, undefined, r => { /* ... */ });
```
I think it's a bit weird to not have a fulfillment handler. But yes, it would be more accurate. I'll change it.

Reply:
Yes. I'm so used to doing this that it doesn't look weird to me any more :smile:.